system/handlers/formcontrols/MultiSelectPanel.cfc

```diff
@@ -14,7 +16,20 @@ component {
 				var props   = presideObjectService.getObjectProperties( objectName );
 
 				for ( var prop in props ) {
-					if ( !( props[ prop ].relationship ?: "" ).reFindNoCase( "to\-many$" ) && !IsTrue( props[ prop ].excludeDataExport ?: "" ) ) {
+					var hasPermission     = true;
```

Comment:
We could/should tweak this to improve performance I think. We _only_ need to do the permissions checking _after_ first doing the straightforward check:

```
if ( !( props[ prop ].relationship ?: "" ).reFindNoCase( "to\-many$" ) && !IsTrue( props[ prop ].excludeDataExport ?: "" ) ) {
...
```

i.e. keep the existing check as it was and then run the additional permission checking within that. Otherwise we end up permission checking fields that would never have made it anyway.